### DISCUSSION TITLE
How to enable PREEMPT_RT instead of PREEMPT_DYNAMIC on Arch 6.12.1 version?

### DISCUSSION THREAD

**Discussion by Tetractys** [zen-kernel/zen-kernel#366] [Q&A]
I installed:
`[I] thor@arch ~/Desktop> uname -a
Linux arch 6.12.1-zen1-1-zen #1 ZEN SMP PREEMPT_DYNAMIC Fri, 22 Nov 2024 16:04:23 +0000 x86_64 GNU/Linux
`
Shouldn't kernel 6.12.x be PREEMPT_RT and not PREEMPT_DYNAMIC?

Where to setup this if possible?
Thanks in advance.

**Comment by heftig** [comment]
It's behind `CONFIG_EXPERT`.

**Comment by Tetractys** [comment]
Thanks. I'll digging for it.
